Pidaq/Lib/paralogging/logInit.py

```diff
@@ -0,0 +1,79 @@
+""" logInit.py
+
+This module contains helper functions for initializing loggers
+for the various paradigm research/testing rigs. The purpose of
+this module is to ensure log format consistency across all code """
```

Comment:
Should `"""` be on a new line? That is what I have been doing, not sure what the proper PEP8 rule for this is, and you would know better than me haha

Reply:
YES - DONE -THANK YOU